go.mod

```diff
@@ -0,0 +1,60 @@
+module github.com/docker/swarmkit
+
+go 1.14
+
+require (
+	code.cloudfoundry.org/clock v1.0.0
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
+	github.com/Microsoft/go-winio v0.4.14
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cloudflare/cfssl v0.0.0-20180323000720-5d63dbd981b5
+	github.com/coreos/etcd v3.3.22+incompatible
+	github.com/coreos/go-systemd v0.0.0-20180511133405-39ca1b05acc7 // indirect
+	github.com/coreos/pkg v0.0.0-20160727233714-3ac0863d7acf // indirect
+	github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
+	github.com/docker/docker v17.12.0-ce-rc1.0.20190319210016-827cb09f8796+incompatible
+	github.com/docker/go-connections v0.4.0
+	github.com/docker/go-events v0.0.0-20170721190031-9461782956ad
+	github.com/docker/go-metrics v0.0.0-20170502235133-d466d4f6fd96
+	github.com/docker/go-units v0.4.0
+	github.com/docker/libkv v0.2.2-0.20180912205406-458977154600 // indirect
+	github.com/docker/libnetwork v0.8.0-dev.2.0.20190726170434-09cdcc8c0eab
+	github.com/dustin/go-humanize v1.0.0
+	github.com/fernet/fernet-go v0.0.0-20180830025343-9eac43b88a5e
+	github.com/gogo/protobuf v1.3.1
+	github.com/google/certificate-transparency-go v1.0.20 // indirect
+	github.com/google/go-cmp v0.4.1 // indirect
+	github.com/gorilla/mux v1.7.4 // indirect
+	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
+	github.com/hashicorp/go-immutable-radix v1.2.0 // indirect
```

Comment:
Might need a `replace` rule for this on (as we still use a fork; 

```
github.com/hashicorp/go-immutable-radix 826af9ccf0feeee615d546d69b11f8e98da8c8f1 git://github.com/tonistiigi/go-immutable-radix.git
```

trying to get rid of that fork though; see the discussion on https://github.com/hashicorp/go-immutable-radix/pull/29

Reply:
We don't need this fork anymore, from Swarmkit's standpoint. The added `Seek` method is unused by Swarmkit or any of its dependencies.

Reply:
When vendored in docker/docker, that fork (and older versions of the related dependencies) will be used, which may be something to take into account if that causes incompatibilities?